src/components/common/Problem.vue

```diff
@@ -29,34 +30,107 @@ export default defineComponent({
 		const _file: Ref<string> = ref(props.file);
 		const renderDiv = ref<HTMLElement>();
 
+		// modified from https://github.com/tserkov/vue-plugin-load-script
+		async function loadScript(src: string) {
+			console.log('loading scripts...', src);
+			// eslint-disable-line no-param-reassign
+			return new Promise<void>(function (resolve, reject) {
+				let shouldAppend = false;
+				let el: HTMLScriptElement | HTMLLinkElement | null;
+				create_el: if (/\.js(?:\??[0-9a-zA-Z=]*)$/.exec(src)) {
+					el = document.querySelector('script[src="' + src + '"]');
+					if (el && el.hasAttribute('data-loaded')) {
```

Comment:
This can be simplified using optional chaining.  Just change it to `if (el?.hasAttribute('data-loaded')) {`.